python/raft/dask/common/comms.py

```diff
@@ -241,46 +326,145 @@ def get_ucx():
     return worker_state("ucp")["ucx"]
 
 
+def _func_destroy_scheduler_session(sessionId, dask_scheduler):
+    """
+    Remove session date from _raft_comm_state, associated with sessionId
+
+    Parameters
+    ----------
+    sessionId : session Id to be destroyed.
+    dask_scheduler : dask_scheduler object
+                    (Note: this is supplied by DASK, not the client)
+    """
+    if sessionId is not None and sessionId in dask_scheduler._raft_comm_state:
+        del dask_scheduler._raft_comm_state[sessionId]
+    else:
+        return 1
+
+    return 0
+
+
+def _func_set_scheduler_as_nccl_root(sessionId, verbose, dask_scheduler):
+    """
+    Creates a persistent nccl uniqueId on the scheduler node.
+
+
+    Parameters
+    ----------
+    sessionId : Associated session to attach the unique ID to.
+    verbose : Indicates whether or not to emit additional information
+    dask_scheduler : dask scheduler object,
+                    (Note: this is supplied by DASK, not the client)
+
+    Return
+    ------
+    uniqueId : byte str
+                NCCL uniqueId, associating the DASK scheduler as its root node.
+    """
+    if verbose:
+        logger.info(
+            msg=f"Setting scheduler as NCCL "
+            f"root for sessionId, '{sessionId}'"
+        )
+
+    if sessionId is None:
+        raise ValueError("sessionId cannot be None.")
+
+    session_state = scheduler_state(
+        sessionId=sessionId, dask_scheduler=dask_scheduler
+    )
```

Comment:
Same with `_func_set_worker_as_nccl_root` and `_func_set_scheduler_as_nccl_root`, these look exactly the same with the only difference being the `get_worker` or `dask_scheduler`. I'm also okay just having these separate functions call a shared function like `_func_set_as_nccl_root(sessionId, process_instance, verbose)`